stl/src/filesystem.cpp

```diff
@@ -966,83 +805,43 @@ struct alignas(long long) _Aligned_file_attrs {
         return _Last_error;
     }
 
-    _GetFileInfoByHandleEx _Get_info_fn;
-    if (_Get_info_fn._Supported()) {
-        constexpr auto _Basic_info_data = __std_fs_stats_flags::_Attributes | __std_fs_stats_flags::_Last_write_time;
-        constexpr auto _Attribute_tag_info_data =
-            __std_fs_stats_flags::_Attributes | __std_fs_stats_flags::_Reparse_tag;
-        constexpr auto _Standard_info_data = __std_fs_stats_flags::_File_size | __std_fs_stats_flags::_Link_count;
-
-        if (_Flags != _Attribute_tag_info_data && _Bitmask_includes(_Flags, _Basic_info_data)) {
-            // we have data FileBasicInfo can fill in, that FileAttributeTagInfo wouldn't exactly fill in
-            FILE_BASIC_INFO _Info;
-            if (!_Get_info_fn(_Handle._Get(), FileBasicInfo, &_Info, sizeof(_Info))) {
-                return __std_win_error{GetLastError()};
-            }
+    constexpr auto _Basic_info_data = __std_fs_stats_flags::_Attributes | __std_fs_stats_flags::_Last_write_time;
+    constexpr auto _Attribute_tag_info_data = __std_fs_stats_flags::_Attributes | __std_fs_stats_flags::_Reparse_tag;
+    constexpr auto _Standard_info_data      = __std_fs_stats_flags::_File_size | __std_fs_stats_flags::_Link_count;

```

Comment:
Weird clang format rules

Reply:
We've configured clang-format to align `=` signs and to stay within 120 columns. When forced to choose, clang-format considers `=` alignment to be lower priority.